modules/framework/galasa-parent/galasa-boot/build.gradle

```diff
@@ -17,6 +17,30 @@ configurations {
 
 version = '0.38.0'
 
+dependencies {
+
+    implementation project (':dev.galasa.framework.maven.repository')
+    embedImplementation project (':dev.galasa.framework.maven.repository.spi')
+
+    implementation platform ('dev.galasa:dev.galasa.platform:0.38.0')
+    implementation 'org.apache.felix:org.apache.felix.scr'
+
+    embedImplementation 'org.apache.felix:org.apache.felix.framework:7.0.5'
```

Comment:
What was the problem with these ones ? We need a story/issue/task to chase these up also ?

Reply:
Task added to the original [story](https://github.com/galasa-dev/projectmanagement/issues/2052) to complete in another change set